pkgs/development/python-modules/anitopy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ python3Packages,
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "anitopy";
+  version = "2.1.1";
+
+  src = fetchFromGitHub {
+    owner = "igorcmoura";
+    repo = "anitopy";
+    rev = "v${version}";
+    hash = "sha256-xXEf7AJKg7grDmkKfFuC4Fk6QYFJtezClyfA3vq8TfQ=";
+  };
+
+  format = "setuptools";
+
+  pythonImportsCheck = [ "anitopy" ];
+
```

Comment:
this package also has tests so you can also enable checks here with `doCheck = true;`